docs/_articles/en/user-guide/vscode-commands.md

```diff
@@ -135,7 +135,7 @@ Read on to learn about the commands available in Salesforce Extension for VS Cod
   - Fetches the most recent debug log from the default org.
   - Executes the CLI command `force:apex:log:get`
   - See [View Apex Debug Logs](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_develop_view_apex_debug_logs.htm).
-- **SFDX: Invoke Apex Tests...**
+- **SFDX: Run Apex Tests...**
```

Comment:
@smitshah2531 Let's remove the ellipsis here

Reply:
Done!!!